src/expressions.md

```diff
@@ -544,6 +547,63 @@ let x: i32 = add(1i32, 2i32);
 let pi: Result<f32, _> = "3.14".parse();
 ```
 
+Function calls may sometimes result in ambiguities about receiver or referent.
+See [UFCS](#unambiguous-function-call-syntax) for how to resolve this.
+
+### Unambiguous Function Call Syntax (UFCS)
```

Comment:
oh that's helpful. with that in mind, maybe I can have a sub-section [here](https://doc.rust-lang.org/nightly/reference/expressions.html#call-expressions) instead of the new section I created? I can describe roughly what you said: "UFCS" confuses the issue, and the RFC didn't add a separate feature. It merely generalized some syntax and turned the then-current syntax into sugar for the fully-qualified function-call syntax. FQFC.

iono @steveklabnik, I'll push some updates, and you can tell me what you think.